### DISCUSSION TITLE
Popstore

### DISCUSSION THREAD

**Discussion by linonetwo** [cosmic-utils/clipboard-manager#108] [Ideas]
Could you upload it to Pop Store? I'm trying PopOS 24 with Cosmic rust version of desktop, and non of clipboard manager on the store is working. And I'm running linux on a very poor machine, so compiling rust code takes very long time, thanks.

**Comment by linonetwo** [comment]
Also, why there is no Pop!_OS in the Install section of Readme? Does it support popos? After I follow [other-distros](https://github.com/cosmic-utils/clipboard-manager?tab=readme-ov-file#other-distros), and click Meta key and search for "clipboard" no new app is shown, why?

**Comment by wiiznokes** [comment]
This is unfortunately the only way to install on pop os until they figuring out a way to package applets.

It is not showing in the app store because it's an applet. You can add it to the panel in cosmic settings.

Keep in mind that you must logout - login for it to works

**Reply by linonetwo** [reply]
Thank you, I find it.

![图片](https://github.com/user-attachments/assets/e645a317-7fe7-4182-af3d-e4bb37fb32b8)

I use Linux on week 2 4, and MacOS on 1 3 5, Win on 6 7, so get to find it today.
